cb-talent-development/employer-style-base
# EM-1483: Typography

**Purpose:**
> We're going to use terms like "style-1" vs H1 to help devs understand what styles they should be applying regardless of what the markup is
> We're overhauling all of the fonts on all of the pages, we're adding all of the stuff we need and documenting it first

**Documentation Updates**
* I updated the pattern library: http://cb-talent-development.github.io/employer-style-base/
    * This is generated with https://livingstyleguide.org/
    * It doesn't handle syntax for mixins and even it's built in sample fonts are not rendering properly
* I took a look at Sassdoc: https://github.com/cb-talent-development/employer-style-base/tree/feat/pattern-library--sassdoc
   * I really like it b/c you can install and document super quickly and easily, but afaik it's not very robust -- it doesn't display colors, or HTML/Haml snippets, it only displays Sass in the documentation it outputs
* I also looked at fractal: https://github.com/cb-talent-development/employer-style-base/tree/feat/pattern-library--fractal
   * And this is what I'd like to go with for generating a pattern library. It allows for really extensive documentation. Take a look at the 24 Ways pattern library: http://bits.24ways.org/
    * But it requires more time to configure, enough so that it's way out of the scope of this story, but I may come back to it during the header stories (if not, we have a pattern library update story in the backlog: https://cb-content-enablement.atlassian.net/browse/EM-644)

**JIRA:**
https://cb-content-enablement.atlassian.net/browse/EM-1483

**Screenshots**
* Before - left, * After - right
![image](https://user-images.githubusercontent.com/2359538/27572354-1616a082-5ad1-11e7-803d-1b2c7ab3a015.png)

**QA Links:**
http://web.employer-2.development.c66.me

Representative set of each page type:
http://web.employer-2.development.c66.me/
http://web.employer-2.development.c66.me/all-products-sitemap
http://web.employer-2.development.c66.me/promotions
http://web.employer-2.development.c66.me/recruiting-solutions
http://web.employer-2.development.c66.me/recruiting-solutions/talent-network/implementation-guide
http://web.employer-2.development.c66.me/employment-screening
http://web.employer-2.development.c66.me/portal/epwhg
http://web.employer-2.development.c66.me/recruiting-solutions/talent-gather
http://web.employer-2.development.c66.me/recruiting-solutions/talent-driver
http://web.employer-2.development.c66.me/recruiting-solutions/supply-and-demand
http://web.employer-2.development.c66.me/recruiting-solutions/job-posting-and-candidate-profile-trends
http://web.employer-2.development.c66.me/recruiting-solutions/job-postings
http://web.employer-2.development.c66.me/recruiting-solutions/resume-database
http://web.employer-2.development.c66.me/recruiting-solutions/small-business-subscription-plans-and-pricing
http://web.employer-2.development.c66.me/dy-test

**How to Verify These Changes**
* Specific pages to visit / Steps to take / Responsive considerations
  * Susan David and I have already looked at various pages so things should look right. Just review the code and dig into anything that looks like it might have broken something that we might not have reviewed or broken something in a way that would be really bad
  * Verify it was okay to delete files
  * View in Safari, which might render text too light to read (but I didn't see any of that)
  * If markup was changed, verify styles are still applying on mobile
  * Check mobile devices


**Relevant PRs/Dependencies:**
https://github.com/cbdr/employer/pull/896

**Additional Information**
